gateway/1.8/modules/ROOT/pages/flex-gateway-k8-ingress-class.adoc

```diff
@@ -52,10 +52,40 @@ kubectl get ingressclasses
 
 To create an ingress routing:
 
-. xref:flex-gateway-publish-apis-local.adoc[Publish an API in Local Mode].
+. xref:flex-local-publish-simple-api.adoc#kubernetes[Publish an API in Local Mode].
 . Define the new API as a Kubernetes Service. +
 For more information about Kubernetes Service definitions, see https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service[Defining a Service^].
-. Create an `Ingress` resource to define routing rules between the `IngressClass` Flex Gateway and the new Kubernetes Service. +
+. Using `kubectl apply`, create an `Ingress` resource to define routing rules between the `IngressClass` Flex Gateway and the new Kubernetes Service:
++
+[source,ssh]
+----
+cat <<EOF | kubectl apply -f -
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: orders-api
+  annotations:
+    flex-gateway.ingress.kubernetes.io/api-instance: ingress-http.gateway
+spec:
+  ingressClassName: my-custom-name
+  rules:
+  - http:
+      paths:
+      - path: /api/orders(/.*)
```

Comment:
Not sure about this, maybe someone just copies this example maybe just put:

```
rules:
  - http: ...
```

or something like that to make obvious that the user needs to put their own config.